### PR TITLE
Fixed Broken Link

### LINKED ISSUE #13162: Broken link in docs/first_steps/first_steps_9.html
At https://docs.bokeh.org/en/latest/docs/first_steps/first_steps_9.html there is currently a broken link to https://github.com/bokeh/bokeh/blob/master/examples/app/sliders.py

I think you renamed "master" -> "main" and it was moved to `examples/server/app/sliders.py`.

Generally keeping docs versions and Github consistent is probably difficult and it might be better to copy all example files and serve them with the docs instead of linking to Github?

I tried using the bug template form, but honestly it doesn't fit for reporting such docs problems.

### PR DESCRIPTION
**Pull request by RajatShenoi**
I've changed the wording slightly and fixed the broken link as mentioned in the issue below.

- [x] issues: fixes #13162

### CODE REVIEW

**Comment by RajatShenoi** [comment]
I don't understand why a check is failing and whether it is failing because of my change. This is my first PR, help would be appreciated. If this solved the issue and doesn't cause any other new issue, consider merging it.

**Comment by mattpap** [comment]
Some tests fail for no good reason, are unrobust, etc. Usually it helps to restart failed tests. I already restarted those, so fingers crossed, it should be all green in a few minutes.

File: docs/bokeh/source/docs/first_steps/first_steps_9.rst [hunk 1]
```diff
@@ -191,8 +191,8 @@ widgets on the left to change the sine wave on the right:
     ></iframe>
     </div>
 
-Find the source code for this Bokeh server app in the
-`Bokeh repository on GitHub <https://github.com/bokeh/bokeh/blob/master/examples/app/sliders.py>`_.
+Find the source code for this Bokeh server app at
+:bokeh-tree:`examples/server/app/slider.py`
```
Comment (bryevdv) [inline docs/bokeh/source/docs/first_steps/first_steps_9.rst]: Thanks @RajatShenoi just one tiny nit, need a full stop at the end of the sentence:
```suggestion
:bokeh-tree:`examples/server/app/slider.py`.
```

**Comment by RajatShenoi** [comment]
I've committed the change including the full stop too. Also, like the previous commit, one check has failed. Please restart that check so that it could be all green again.

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
The one failure is clearly spurious, merging now. Thanks @RajatShenoi

**Pull request merged**